commitlintplugins.test.ts

```diff
@@ -165,6 +165,7 @@ test('commit-hash-alone1', () => {
         "foo: this is only a title" + "\n\n" + 
         `https://github.com/nblockchain/conventions/commit/3ee07243edc30604088a4b04ca525204ea440710`;
     let commitHashAlone1 = runCommitLintOnMsg(commitMsgWithCommitUrl);
+    console.log('HERE ===>' + commitHashAlone1.stdout)
```

Comment:
@realmarv that's my fault, I'm gonna fix it in master and after that you can rebase